unified-runtime/source/adapters/native_cpu/context.cpp

```diff
@@ -72,33 +71,21 @@ urContextGetInfo(ur_context_handle_t hContext, ur_context_info_t propName,
 }
 
 UR_APIEXPORT ur_result_t UR_APICALL urContextGetNativeHandle(
-    ur_context_handle_t hContext, ur_native_handle_t *phNativeContext) {
-  std::ignore = hContext;
-  std::ignore = phNativeContext;
-  DIE_NO_IMPLEMENTATION
-}
+    ur_context_handle_t /*hContext*/,
+    ur_native_handle_t * /*phNativeContext*/){DIE_NO_IMPLEMENTATION}
```

Comment:
The formatting here looks weird, likely as a result of `DIE_NO_IMPLEMENTATION` being defined to include the semicolon. Could you include a redundant semicolon here (and below) to get it formatted in a more sensible way? Also okay if you want to take out the semicolon from `DIE_NO_IMPLEMENTATION`'s definition if you like, but not required.